### DISCUSSION TITLE
January 18th 2024 Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#21041] [Community Call]
Agenda:

**Comment by SydneyhSmith** [comment]
- VSCode stable update (Sydney)
- PowerShell 7.5 preview (Dongbo)
- Improvements to cmdlets (Steve)
- PSResourceGet servicing release (Sydney)
- March community call (Steven)
- Change in WAM (Damien)
- Auth command AzPS (Damien)
- Docs update (Sean)
- WinGet Command not found (@crutkas)
- Community Demos

**Comment by JustinGrote** [comment]
I would like to do a 5 min demo of the latest ModuleFast 0.1.0 Release. Thanks! :)
https://github.com/JustinGrote/ModuleFast/releases/tag/v0.1.0

**Comment by rlwakefield** [comment]
Is there still availability for demos? I ask because I have been working on a new version of the original Test-PendingReboot.ps1 script created by Adam The Automator and enhancing it as well as clarifying it into something of my own (for work purposes). So I would love to get some feedback from people. (NOTE: I haven't put it on Github yet, but I will definitely work on doing so.)

I am definitely a little nervous to do so as this would be my very first time, but I am quite proud of what I have accomplished so far.

Thanks!

**Reply by theJasonHelmick** [reply]
@rlwakefield - Ryan, love to have you show your demo tomorrow if we have time. I'll add you to the list.  No pressure, you can be first on the list next month - your choice.  for now, I've added you to the Agenda for Demos tomorrow - Feel free to ping me in the meeting if wish to move to next month.

Thank you!

**Reply by JustinGrote** [reply]
@rlwakefield hey great job with the demo! That looked super useful.

**Comment by StartAutomating** [comment]
If there's time, I'd love to briefly show off implicit interpretation of other languages within PowerShell.

I think this will be a decent game changer for the community, since it will help break down the walls between PowerShell's ecosystem and other languages.

**Reply by theJasonHelmick** [reply]
Sure James -- might want to plan for next month as I think we have a full schedule -- but if time permits no problem :)
